test/local-fs.spec.ts

```diff
@@ -114,6 +115,14 @@ describe(`the local filesystem implementation`, () => {
                 .then(() => expect(fs.loadTextFile(fileName)).to.eventually.equals(newContent));
         });
 
+        it(`ignores events from ignored dir in watcher`, () => {
```

Comment:
> I would rather starting with a single error event with 0 arguments. when a user gets this event she should drop all local caches and re-read the file system

it is not un-synced, basically that means that watcher send event which isn't expected, because path is under ignoring mask (we had that case) or outside of watch root (we don't have that case, but I am wondering).
Then local-fs tries to read that path and gets rejection, because it is ignored, no need to re-sync all tree, nothing important happened with this event.